pyscriptjs/tests/integration/test_splashscreen.py

```diff
@@ -97,3 +97,44 @@ def myFunc(*args, **kwargs):
 
         assert self.page.locator("py-splashscreen").count() == 0
         assert "Python exception" in self.console.error.text
+
+    def test_splashscreen_disabled_option(self):
+        self.pyscript_run(
+            """
+            <py-config>
+                [splashscreen]
+                disabled = true
```

Comment:
The key being used is `enabled` as written in documentation and code. But here, the key being used is `disabled`. So ideally, this test shouldn't even pass, right?

Reply:
right, I think it might be passing because the loader disappeared too quickly, fixed the test 😄

Reply:
Maybe we can add another test that ensures we do have a splashscreen? I commented the same below since I saw this later. So resolving this thread at least.